libraries/psibase/native/src/SystemContext.cpp

```diff
@@ -57,7 +61,8 @@ namespace psibase
       std::lock_guard<std::mutex> lock{impl->mutex};
       if (impl->systemContextCache.empty())
       {
-         auto result = std::make_unique<SystemContext>(SystemContext{impl->db, impl->wasmCache});
+         auto result = std::make_unique<SystemContext>(
+             SystemContext{impl->db, impl->wasmCache, {}, impl->watchdogManager});
          impl->allSystemContexts.push_back(result.get());
```

Comment:
This looks sketchy. Couldn't this lead to UB if the unique ptr is destroyed and the linearMemorySpan is called?

Reply:
Yes, though that never happens. Also, note the comment on the declaration of allSystemContexts.

Reply:
This is pre-existing code that is not modified by this PR.